### PR TITLE
Rewrite vWii Modding Guide

### PR DESCRIPTION
**Pull request by VannyBuns**
Made it similar to the Tiramisu guide and added a Finalizing Setup

### CODE REVIEW

File: docs/user-guide/tiramisu/nand-backup.md [hunk 1]
```diff
@@ -22,7 +22,7 @@ In case anything should go wrong in the later process and your Wii U ends up bri
 1. Press the A button to start the dumping process.
 1. When the process completed, power off your Wii U, take your SD Card out of the Wii U and plug it into your PC.
 1. The files will appear on the root of the SD card.
-1. To make sure you don't lose the files, copy the `slc.bin`, `slccmpt.bin`, `seeprom.bin`, `otp.bin` (and if you chose to go with a full backup, `every mlc.bin.part`) file to your computer.
+1. To make sure you don't lose the files, copy the `slc.bin`, `slccmpt.bin`, `seeprom.bin`, `otp.bin` (and if you chose to go with a full backup,every `mlc.bin.part`) file to your computer somewhere safe (OneDrive, Google Drive, .etc).
```
Comment (Lazr1026) [inline docs/user-guide/tiramisu/nand-backup.md:25]:
```suggestion
1. To make sure you don't lose the files, copy the `slc.bin`, `slccmpt.bin`, `seeprom.bin`, `otp.bin` (and if you chose to go with a full backup,every `mlc.bin.part`) file to your computer somewhere safe (OneDrive, Google Drive, etc).
```

File: docs/user-guide/vwii/finalizing-setup.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+### Additional Homebrew Apps
+
+- **SaveGame Manager GX** allows you to backup and restore your saves and Miis, especially saves that can't usually be copied.
+- **Nintendont** is a virtual machine that allows you to play Gamecube games.
+- The **Open Shop Channel** is an appstore where you can download more apps.
+
+### Additional Homebrew Apps - What You Need
+
+- The latest release of [SaveGame Manager GX](https://hbb1.oscwii.org/api/v3/contents/SaveGame_Manager_GX/SaveGame_Manager_GX.zip)
+- The `loader.dol`, `meta.xml` and `icon.png` of [Nintendont](https://github.com/GaryOderNichts/Nintendont)
```
Comment (Lazr1026) [inline docs/user-guide/vwii/finalizing-setup.md:16]: Can't we just link the one on OSC? Gary's changes have been merged upstream afaik

File: docs/user-guide/vwii/finalizing-setup.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+- The `Homebrew Browser.zip` of the [Open Shop Channel](https://oscwii.org/library/app/homebrew_browser)
```
Comment (Lazr1026) [inline docs/user-guide/vwii/finalizing-setup.md:17]: Can probably just directly link to the download like the SGMGX

**Review by Lazr1026 — CHANGES_REQUESTED** [review]
side note: good job learning how to github

File: docs/user-guide/vwii/finalizing-setup.md (hunk 2 above)
Comment (VannyBuns) [reply docs/user-guide/vwii/finalizing-setup.md:16]: So is it the same one that fixed the black screen controller issue?

File: docs/user-guide/vwii/finalizing-setup.md (hunk 3 above)
Comment (VannyBuns) [reply docs/user-guide/vwii/finalizing-setup.md:17]: For which one

File: docs/user-guide/vwii/finalizing-setup.md (hunk 2 above)
Comment (Lazr1026) [reply docs/user-guide/vwii/finalizing-setup.md:16]: Actually, im not sure anymore. I was told that it was merged upstream but i do not see any PR from Gary to the Nintendont repo.

Comment (Lazr1026) [reply docs/user-guide/vwii/finalizing-setup.md:16]: I suppose we could

**Pull request merged**